fmn/consumer/backend.py

```diff
@@ -26,6 +23,8 @@
 import fmn.consumer.fmn_fasshim
 from fedmsg_meta_fedora_infrastructure import fasshim
 
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
I was told once that this should be run as early as possible, is it not the case?